Ginger/GingerCoreNET/Application Models/Execution/POM/POMExecutionUtils.cs

```diff
@@ -332,6 +332,11 @@ public ElementInfo AutoUpdateCurrentPOM(Common.InterfacesLib.IAgent currentAgent
                 {
                     return null;
                 }
+                else
```

Comment:
Add it where its actually going for POM update

Reply:
done